### ISSUE TITLE
Upgrade from C100 -> C101 on `Win x64` via`1.39.x - Nightly` (Checking C101)

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#22312]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromoium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `cffkpbalmllkdoenhmdmpbkajipdjfam`: `rs-ABPFilterParserData.dat` & `regional_catalog.json` (AdBlock)
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `jicbkmdloagakknpihibphagfckhjdih`: `speedreader-updater.dat`
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers. We basically want to make sure that components are working as they regressed in C98 when we first merged into `master`.

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Rewards
    - [x] Wallet balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Wallet panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 11 x64` using the following build(s):

```
Brave | 1.39.57 Chromium: 101.0.4951.34 (Official Build) nightly (64-bit)
--- | ---
Revision | 8f93af5d759634c5256b65997dedd138ed60992f-refs/branch-heads/4951@{#710}
OS | Windows 11 Version 21H2 (Build 22000.613)
````

Ensured there's no outbound third party connections occurring on startup or when a user opens a NTP as per the following:

![startupConnections](https://user-images.githubusercontent.com/2602313/163480737-d68fe389-09ab-40e9-8183-1149cd20c8c4.png)

Ensured that there's no new components that might have been added in `C101` that should be removed as per the following:

**`100.0.4896.88`** | **`101.0.4951.34`**
---------------------|----------------------
![image](https://user-images.githubusercontent.com/2602313/163480913-df4b1380-cc6b-4b0e-9379-f0a443921496.png) | ![101 0 4951 34](https://user-images.githubusercontent.com/2602313/163480919-ce89be04-f1ee-4867-829c-c5322bf6d13b.png)

**Comment by MadhaviSeelam** [comment]
Verification `Passed` using

Brave | 1.39.57 Chromium: 101.0.4951.34 (Official Build) nightly (64-bit)
-- | --
Revision | 8f93af5d759634c5256b65997dedd138ed60992f-refs/branch-heads/4951@{#710}
OS | Windows 11 Version 21H2 (Build 22000.593)

Component Updater
100.0.4896.88|101.0.4951.34
---|---
<img width="950" alt="Components_c101" src="https://user-images.githubusercontent.com/98358127/163497540-d3670579-520d-472c-a69b-fb403ec7acbf.png">|<img width="950" alt="components_C100" src="https://user-images.githubusercontent.com/98358127/163497598-59571e1d-29b3-4b71-ac64-33da19f0ec39.png">
